scripts/ci/test-operator

```diff
@@ -110,7 +110,13 @@ trap_add_exit "delete_objects_incluster $ABS_BUNDLE_PATH $NAMESPACE"
 
 # Create logs for operator debugging.
 LOG_FILE="${TMP}/info.log"
-trap_add_exit "log_operator_state $LOG_FILE OBJECTS $NAMESPACE; cat $LOG_FILE"
+trap_add_exit "log_operator_state $LOG_FILE OBJECTS $NAMESPACE; \
+  fold_start operatorState operatorLogs; cat $LOG_FILE; fold_end operatorState"
```

Comment:
These functions only work in a travis environment. To make this script extensible to AWS, wrap `fold_start ...;` in conditionals:
`[[ -n "$TRAVIS_PULL_REQUEST" ]] && fold_start ...`